api/Navigator.json

```diff
@@ -320,15 +331,6 @@
             },
             "samsunginternet_android": {
               "version_added": null
-            },
-            "uc_android": {
-              "version_added": null
-            },
-            "uc_chinese_android": {
-              "version_added": null
```

Comment:
I generally prefer to keep these

Reply:
Then should we update it everywhere? I copied from the SW and these did not appear anywhere.